coldfront/core/resource/models.py

```diff
@@ -53,8 +85,21 @@ def __str__(self):
     class Meta:
         ordering = ['name', ]
 
-
 class Resource(TimeStampedModel):
+    """ A resource is something a center maintains and provides access to for the community. Examples include Budgetstorage, Server, and Software License. 
+    
+    Attributes:
+        parent_resource (Resource) = used for the Cluster Partition resource type as these partitions fall under a main cluster
```

Comment:
Fix typo.. these should to have a ':' instead of an '='. 

```
WARNING  -  mkdocstrings_handlers: coldfront.core.resource.models.Resource: Failed to get 'name: description' pair from 'parent_resource (Resource) = used for
            the Cluster Partition resource type as these partitions fall under a main cluster'
```